compiler/circlechef/proto/circlechef.proto

```diff
@@ -14,13 +14,15 @@ package circlechef;
 
 // This enum value corresponds to TensorType in TensorFlow Lite schema
 enum TensorType {
+  UINT4 = -1;
```

Comment:
Just curiosity, is there any reason for using a negative value only on `UNIT4`?

Reply:
We can give any non-occupied value for UINT4.
This value doesn't depend on circle-schema UINT4.
But can be curious why values in this file and schema be different.
So, just followed that of circle-schema, which is -1

Reply:
Anyway, if someone gives (strong) opinion on specific value, I can follow :)